schema/artwork/index.js

```diff
@@ -277,6 +277,12 @@ const ArtworkType = new GraphQLObjectType({
         type: GraphQLBoolean,
         resolve: ({ price_hidden }) => price_hidden,
       },
+      is_price_range: {
+        type: GraphQLBoolean,
+        resolve: (artwork) => {
+          return new RegExp(/\-/).test(artwork.price);
```

Comment:
I think you can use that bracket format from the other field (where matt said it's unnecessary) here if you wanna be clever
```
      is_price_range: {
        type: GraphQLBoolean,
        resolve: ({ price }) => {
          return new RegExp(/\-/).test(price);
```